packages/core/src/createProps.ts

```diff
@@ -219,14 +219,14 @@ export const getPropTable = (): [string, string, string, CCSSParser?, CCSSParser
 }
 
 const getPropTableObject = () => {
-    const tableObject: CCSSProps = {}
+    const tableObject = {} as CCSSProps
```

Comment:
Just a small comment here:
This `cons c: SomeType = {}` differs from `const c = {} as SomeType` because, on the first sample, it is expected that `Object` will have type `SomeType` and this is not true, because `{}` doesn't have any expected properties from `SomeType` type. The second sample we tell TS that `{}` actually has `SomeType` type, so TS will treat it like so.

Reply:
Use `Partial`